Examples/ArcGISToolkitExamples/Misc/OptionsTableViewController.swift

```diff
@@ -0,0 +1,86 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+/// A basic interface for selecting one options from a list,
+/// showing the checkmark accessory for the selected cell.
+class OptionsTableViewController: UITableViewController {
```

Comment:
FYI: There are some substantive issues with this view controller. For example, rows aren't highlighted when touched. I plan on fixing that and making other improvements to this view controller at some point.

Reply:
Using UIAlertViewController as an action sheet.